16-google/incident.ts

```diff
@@ -0,0 +1,117 @@
+/**
+ * An exploration which demonstrates packet loss and network congestion
+ * after clusters' cluster management software are de-scheduled.
+ * 
+ * This exploration exists to prove the design of the Database, Cluster 
+ * Management Software, and BGP stages appropriately mock the architecture 
+ * and problems listed in the incident report, particularly the failures 
+ * described for Google's Cloud Interconnect service.
+ * 
+ */
+
+import {
+  metronome,
+  simulation,
+  stats,
+  eventSummary,
+  stageSummary,
+  Timeout,
+  eventCompare
+} from "../../src";
+import { Cluster, Server } from "./database"
+import { ClusterManagementSoftware } from "./cms";
+import { BGP } from "./bgp";
+
+//Cluster 1
+//Server 1
+//Server 2
+//Server 3
+const s1 = new Server();
+const s2 = new Server();
+const s3 = new Server();
+const db1 = new Cluster([s1, s2, s3]);
+
+//Cluster 2
+//Server 4
+//Server 5
+//Server 6
+const s4 = new Server();
+const s5 = new Server();
+const s6 = new Server();
+const db2 = new Cluster([s4, s5, s6]);
+
+//BGP 1 and 2
```

Comment:
Are these BGPs representing regions? If so, consider renaming them to region1 and region2. Not a requirement, just a possible suggestion that you can decide if its valuable or not.

Reply:
If not, ignore me. It just depends on what you wanted the things below the BGP to represent.